syntax = "proto3";
package test;

import "msg.proto";

service AsyncScheduleService {
  rpc SlowUnary(Request) returns (Reply) {}
  rpc DepUnary(Request) returns (Reply) {}
  rpc BidiStream(stream Request) returns (stream Reply) {}
}
